src/app/services/xml-parsers/index.ts

```diff
@@ -27,11 +27,14 @@ export class ParserRegister {
         if (nels.includes(tagName)) {
             return 'evt-named-entities-list-parser';
         }
+        const quote = ['quote', 'cit', 'ref', 'seg', 'div'];
```

Comment:
I honestly don't like this approach, in particular for elements like `seg` or `div` which are very generic and not always used for quotes. I would define a parser dedicated for each element and insert inside of it the check: if it's a quote use quote parser. @szenzaro @giuliac89 what do you think?

Reply:
I could either do that or add a disambiguation parser to distinguish from ref, seg, div that aren't sources or analogues and the ones that are. This because the general parser only discriminates on tagName basis which is not sufficient for this purpose. 
That external disambiguation parser we can image (and which was a first solution you can retrieve in first commits) will result in much duplicated code though.